frontend/packages/schema-editor/src/components/SchemaInspector/ItemRestrictions/EnumList/EnumField/EnumField.tsx

```diff
@@ -42,7 +42,7 @@ export const EnumField = ({
   const onKeyDown = (e: KeyboardEvent<HTMLInputElement>) =>
     e?.key === 'Enter' && onEnterKeyPress && onEnterKeyPress();
 
-  const label = t('schema_editor.textfield_label', { id: `${baseId}-enum-${value}` });
+  const label = t('schema_editor.textfield_label', { id: `${index}-enum-${value}` });
```

Comment:
Dette bør være en lesbar tekst. `value` trenger heller ikke å være med i ledeteksten. Jeg foreslår at vi endrer teksten med nøkkelen `schema_editor.textfield_label` til `Gyldig verdi nummer {{index}}` (nå står det `Rad i listen med gyldige verdier med id {{ id }}`). Da kan vi endre denne linjen slik:
```suggestion
  const label = t('schema_editor.textfield_label', { index });
```
Nøkkelen `textfield_label` er for øvrig heller ikke spesifikk nok. Hva med å endre den til f.eks. `enum_value`?